css/registrationFrom.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="bg">
+<head>
+    <title>Регистрация</title>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <link rel="stylesheet" type="text/css" href="registrationForm.css">
+</head>
+</head>
+<body>
+    <form method="POST">
+        <h4>РЕГИСТРАЦИЯ</h4>
+        <input type="text" class="registration-fields" placeholder="E-mail">
+        <br>
```

Comment:
You may want to use margins instead of new lines.

Reply:
This will make the code cleaner and also will give you more control over the presentation of the html elements.